rocAL/rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -246,7 +246,7 @@ rocalJpegFileSource(
         info.set_tensor_layout(RocalTensorlayout::NHWC);
         info.set_max_shape();
         output = context->master_graph->create_loader_output_tensor(info);
-        auto cpu_num_threads = context->master_graph->calculate_cpu_num_threads(shard_count);
+        auto cpu_num_threads = context->master_graph->calculate_cpu_num_threads(1);
```

Comment:
why shard_count is not used here?

Reply:
This API has no param called shard_count. The value here is supposed to be 1, I think it was set to shard count due to some merge conflicts.